config-model/src/main/java/com/yahoo/vespa/model/search/NodeFlavorTuning.java

```diff
@@ -99,4 +100,11 @@ private void tuneSearchReadIo(ProtonConfig.Search.Mmap.Builder builder) {
         }
     }
 
+    private void tuneWriteFilter(ProtonConfig.Writefilter.Builder builder) {
+        // "Reserve" 1GB of memory for other processes running on the content node (config-proxy, cluster-controller, metrics-proxy)
+        double defaultMemoryLimit = new ProtonConfig.Writefilter(new ProtonConfig.Writefilter.Builder()).memorylimit();
+        double scaledMemoryLimit = ((nodeFlavor.getMinMainMemoryAvailableGb() - 1) * defaultMemoryLimit) / nodeFlavor.getMinMainMemoryAvailableGb();
```

Comment:
Consider using a named constant for the 1GB reserved, rather than '1'.

Reply:
Fixed